frontend/src/app/[locale]/search/loading.tsx

```diff
@@ -1,14 +1,16 @@
+import { useTranslations } from "next-intl";
 import React from "react";
 
 import Spinner from "src/components/Spinner";
 
 export default function Loading() {
-  // TODO (Issue #1937): Use translation utility for strings in this file
+  const t = useTranslations("Loading");
```

Comment:
```suggestion
  unstable_setRequestLocale("en");

  const t = useTranslations("Loading");
```

Reply:
Making these changes causes the following error in Storybook:
![image](https://github.com/user-attachments/assets/69efedcb-c921-4247-bb68-d9c47a903a35)

Looks like this might not be needed at the component level?
https://next-intl-docs.vercel.app/docs/getting-started/app-router/with-i18n-routing#add-unstable_setrequestlocale-to-all-layouts-and-pages